src/RokuDeploy.ts

```diff
@@ -507,8 +507,23 @@ export class RokuDeploy {
             archive: '',
             mysubmit: 'Convert to squashfs'
         });
-
-        let results = await this.doPostRequest(requestOptions);
+        let results;
+        try {
+            results = await this.doPostRequest(requestOptions);
+        } catch (error) {
+            if ((error as any)?.code === 'HPE_INVALID_CONSTANT') {
```

Comment:
We could use a comment here explaining what we're doing. Probably a more detailed comment would be best, cuz this is a weird one. :D

Reply:
👍

Reply:
My comment is a bit wordy. Probably want to nitpick it a bit.